spec/fixtures/invalid.txt

```diff
@@ -0,0 +1 @@
+invalid
```

Comment:
Can someone help me understand what this is 🤔

Reply:
This is a test fixture. It's basically a file that we use in our tests. This one is used to test that the logo upload rejects files that are not PNGs or SVGs.